tests/integration/test_api.py

```diff
@@ -33,6 +35,23 @@ def test_experiment_flow():
 
     check_experiments(experiment, experiment_new)
 
+    with NamedTemporaryFile() as file1:
+        with NamedTemporaryFile() as file2:
+            with NamedTemporaryFile() as file3:
+                with NamedTemporaryFile() as file4:
+                    expected_files = [file1.name, file2.name, file3.name, file4.name]
+                    experiment.upload_file(file1.name)
+                    experiment.upload_file(file2.name)
+                    experiment.upload_file(file3.name)
+                    experiment.upload_file(file4.name)
+
+                    experiment.remove_files(
+                        files=[file1.name.split("/")[-1], file2.name.split("/")[-1]]
+                    )
+                    assert sorted([item[0] for item in experiment.files]) == sorted(
+                        [file3.name.split("/")[-1], file4.name.split("/")[-1]]
+                    )
+
```

Comment:
A separate test on the new function will be good.

Reply:
I couldn't understand which function you are referring to. @str-anger could you point me to that?

Reply:
`remove_files` is a new method, you've added a test to an integration workflow, but it would be nice to have tests covering the function and its flow separately.